databricks/labs/doc_qa/evaluators/templated_evaluator.py

```diff
@@ -15,7 +15,7 @@
 
 
 logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger(__name__.split(".")[0])
+logger = logging.getLogger(__name__.split(".")[-1])
```

Comment:
This hints that you should inject the logger and not look it up 😉

Reply:
Good point! Updated.